.github/workflows/ci.yml

```diff
@@ -7,42 +7,42 @@ on:
 jobs:
   build:
     name: Build
+
     strategy:
       fail-fast: false
       matrix:
         os:
           - ubuntu-latest
           - macos-latest
-        ocaml-compiler:
-          - "4.12"
-          - "4.13"
-          - "4.14"
```

Comment:
OCaml 4.13 or later is supported on Windows. It could be improved in the future, but honestly, testing on 4.14 is good enough.

Reply:
Thank you!

How about keeping this array as the following singleton?

```yaml
ocaml-compiler:
- "4.14"
```

I think this will ease the future addition of other versions.

Reply:
Sure!